plugins/python/io/NumpyReader.cpp

```diff
@@ -239,6 +297,13 @@ void NumpyReader::addArgs(ProgramArgs& args)
     m_orderArg = &args.add("order", "Order of dimension interpretation "
         "of the array.  Either 'row'-major (C) or 'column'-major (Fortran)",
         m_order);
+
+    args.add("module", "Python module containing the function to run",
+        m_args->module, "");
```

Comment:
You don't need the empty string defaults.

Reply:
addressed by fddec900a5c59978842bd541e53784e578215084